context/app/static/js/components/Detail/entityHeader/EntityHeaderItem/EntityHeaderItem.jsx

```diff
@@ -14,7 +14,7 @@ function EntityHeaderItem({ text }) {
 }
 
 EntityHeaderItem.propTypes = {
-  text: PropTypes.string,
+  text: PropTypes.oneOfType(PropTypes.string, PropTypes.arrayOf(PropTypes.string)),
```

Comment:
Not sure exactly how this is used, but could we say it's always an array, and perhaps the length is just one? Since both array and string can be subscripted, and have a length, I'd really like to avoid unioning these types.